lib/ohdear_health_check/configuration.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module OhdearHealthCheck
+  class Configuration
+    SETTINGS = %i[checks].freeze
+
+    attr_accessor(*SETTINGS)
+
+    def initialize
+      clear!
+      add_default_checks
+    end
+
+    def add_check(name, block, success_message = nil, error_message = nil)
+      @checks << Check.new(name, block, success_message, error_message)
+    end
+
+    def clear!
+      @checks = []
+    end
+
+    private
+
+    def add_redis
+      return unless defined?(Redis)
+
+      add_check :redis, -> { Redis.new.ping }, 'Redis is up', 'Redis is down'
+    end
```

Comment:
If we're ready to add a few more of those, it might be smart to put those in their own files and include them here